### PR TITLE
add very cool feature

### PR DESCRIPTION
**Pull request by gerysimons**
Some descr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # PR-test
 test
+Add new feature
```
Comment (MrGrey10) [inline README.md]: Its cool!

**Review by MrGrey10 — COMMENTED** [review]
write "!" symbol in the end

File: README.md (hunk 1 above)
Comment (arthurspl) [reply README.md]: ok

**Comment by gerysimons** [comment]
add fixes, approve please

**Review by MrGrey10 — APPROVED** [review]
Excellent

**Pull request merged**